### PR TITLE
etcdctl:  support query last-rev or first-rev only of specified prefix

### PR DESCRIPTION
**Pull request by xrmzju**
support query last-rev or first-rev only of specified prefix in get cmd of etcdctl

### CODE REVIEW

File: etcdctl/ctlv3/command/get_command.go [hunk 1]
```diff
@@ -112,44 +123,56 @@ func getGetOp(args []string) (string, []clientv3.OpOption) {
 		opts = append(opts, clientv3.WithRange(args[1]))
 	}
 
-	opts = append(opts, clientv3.WithLimit(getLimit))
+	if getLimit != 0 {
```
Comment (cfc4n) [inline etcdctl/ctlv3/command/get_command.go:126]: Can `getLimit`  be negative?

Comment (xrmzju) [reply etcdctl/ctlv3/command/get_command.go:126]: seems we did not care much about this...all options provided by clientv3 uses int64
```go
func WithLimit(n int64) OpOption ...
func WithRev(rev int64) OpOption ...
```

Comment (cfc4n) [reply etcdctl/ctlv3/command/get_command.go:126]: Why not `if getLimit > 0` ?

Comment (xrmzju) [reply etcdctl/ctlv3/command/get_command.go:126]: yeah, u are right..

**Pull request closed without merging**